portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACSettings.macro

```diff
@@ -40,6 +40,16 @@ definition {
 			value1 = "${pageViewsUsageValue}");
 	}
 
+	macro assertPlanDetails {
+		key_planName = "${planName}";
```

Comment:
Hi @AfonsoHSantos , could you please declare the variable: 
`var key_planName = "${planName}";`

Reply:
fixed